src/GuiBase/Viewer/CameraInterface.hpp

```diff
@@ -67,6 +68,9 @@ namespace Ra
 
             virtual void update( Scalar dt ) {}
 
+
+            static const Engine::Camera& getCameraFromViewer(QObject *v);
+

```

Comment:
I'm also not really happy about that but it does the work ... Need a more indepth refactoring to be able to access the current camera from different location in the software. But is this really a need or an ease ?